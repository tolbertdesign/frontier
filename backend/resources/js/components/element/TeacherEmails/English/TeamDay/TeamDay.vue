<template>
  <p>
    To: School Families
    <br>
    When: Team Day 1
    <br>
    Subject: Help our class!
    <br><br>
    {{ props.event_name }} Families,
    <br><br>
    Our class is having a BLAST preparing for the  {{ props.event_name }} coming up soon! We've also begun
    learning how to choose to live with character every day through Booster's brand new character theme,
    MindSpark: Wild West. While they're teaming up with the MindSpark Kids to compete in character duels,
    your students will learn five 21st-century character traits that will help strengthen themselves and our world.
    Make sure to ask your students what they've learned each day, and together you can watch
    the story unfold on <a :href="`https://${props.link}`">{{ props.link }}</a>!
    <br><br>Remember, we need your help connecting to sponsors who can make a pledge to help our school!
    Please make sure all pledges for the  {{ props.event_name }} are entered on <a :href="`https://${props.link}`">{{ props.link }}</a>.
    The more pledges we get, the more incentives our class earns on our Pledge-O-Meter.
    <br><br>Thank you for your generosity and support of our school. If you have any questions, please let me know.
    <br><br>Have a great day!
    <br><br>P.S. Make sure to register your child on <a :href="`https://${props.link}`">{{ props.link }}</a>.
    Search for your school or use our school registration code:
    <br><strong>{{ props.registration_code }}</strong><br>
  </p>
</template>

<script>
export default {
  name: 'TeamDay',
  status: 'prototype',
  release: '1.0.0',
  props: {
    props: {
      type: Object,
      default: () => {},
    },
  },
}
</script>
