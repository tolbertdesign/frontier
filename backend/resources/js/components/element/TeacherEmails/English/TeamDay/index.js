import TeamDay from './TeamDay.vue'

export default TeamDay
