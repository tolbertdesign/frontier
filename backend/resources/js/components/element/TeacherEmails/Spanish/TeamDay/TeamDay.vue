<template>
  <p>
    Para: Familias de la Clase
    <br>Cuando: Dia de equipo uno
    <br>Tema: ¡Ayuda nuestra clase!
    <br><br>{{ props.event_name }} Familias,
    <br><br>¡Nuestra clase se esta divirtiendo con preparación para el <strong>{{ props.event_name }}</strong>
    que viene pronto! También hemos comenzado a aprender cómo elegir vivir con un buen carácter todo los días a
    través de los ensenajes del nuevo tema de Booster: Mindspark Wild West. Ellos van acompañar al equipo
    de MindSpark para completar duelos de carácter, sus estudiantes aprenderán cinco rasgos de carácter
    del siglo XXI que ayudarán a fortalecerse a sí mismos y a nuestro mundo. ¡Aseguerse de preguntarles a
    sus estudiantes de lo que han estado aprendiendo cada día, y juntos pueden ver cómo se desarrolla
    la historia en <a :href="`https://${props.link}`">{{ props.link }}</a>!
    <br><br>Recuerde, necesitamos su ayuda para conectarse con los patrocinadores que puede hacer una donación
    para ayudar nuestra escuela! Asegurese de que todas las donaciones para el
    <strong>{{ props.event_name }}</strong> están en <a :href="`https://${props.link}`">{{ props.link }}</a>.
    Cuántas mas donaciones que recibimos, más incentivos podemos obtener en la clase en nuestro Pledge-O-Meter.
    <br><br>Muchas gracias por su generosidad y apoyo para ayudar nuestra escuela. Si usted tiene preguntas, por favor déjeme saber.
    <br><br>¡Que tenga un buen día!
    <br><br>PD: Asegurese de registrar su estudiante en <a :href="`https://${props.link}`">{{ props.link }}</a>.
    Utilice nuestro código de registro escolar: <strong>{{ props.registration_code }}.</strong><br>
  </p>
</template>

<script>
export default {
  name: 'TeamDay',
  status: 'prototype',
  release: '1.0.0',
  props: {
    props: {
      type: Object,
      default: () => {},
    },
  },
}
</script>
