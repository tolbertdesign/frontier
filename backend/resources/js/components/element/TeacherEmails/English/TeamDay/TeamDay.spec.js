import { shallowMount, createLocalVue } from '@vue/test-utils'
import TeamDay from './TeamDay.vue'

const localVue = createLocalVue()

const props = {
  event_name: 'event name',
  registration_code: 'registration code',
  due_date: 'due date',
  link: 'example.com',
  class_pledge_total: '500',
}

describe('TeamDay', () => {
  it('Snapshop test for TeamDay', () => {
    const wrapper = shallowMount(TeamDay, {
      localVue,
      propsData: {
        props: props,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
