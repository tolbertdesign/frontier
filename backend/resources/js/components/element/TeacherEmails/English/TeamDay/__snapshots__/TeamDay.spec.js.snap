// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamDay Snapshop test for TeamDay 1`] = `
<p>
  To: School Families
  <br>
  When: Team Day 1
  <br>
  Subject: Help our class!
  <br><br>
  event name Families,
  <br><br>
  Our class is having a BLAST preparing for the event name coming up soon! We've also begun
  learning how to choose to live with character every day through Booster's brand new character theme,
  MindSpark: Wild West. While they're teaming up with the MindSpark Kids to compete in character duels,
  your students will learn five 21st-century character traits that will help strengthen themselves and our world.
  Make sure to ask your students what they've learned each day, and together you can watch
  the story unfold on <a href="https://example.com">example.com</a>!
  <br><br>Remember, we need your help connecting to sponsors who can make a pledge to help our school!
  Please make sure all pledges for the event name are entered on <a href="https://example.com">example.com</a>.
  The more pledges we get, the more incentives our class earns on our Pledge-O-Meter.
  <br><br>Thank you for your generosity and support of our school. If you have any questions, please let me know.
  <br><br>Have a great day!
  <br><br>P.S. Make sure to register your child on <a href="https://example.com">example.com</a>.
  Search for your school or use our school registration code:
  <br><strong>registration code</strong><br></p>
`;
